test/router_SUITE.erl

```diff
@@ -937,6 +939,311 @@ us915_join_disabled_cf_list_test(Config) ->
+                <<"port">> => fun erlang:is_integer/1,
+                <<"devaddr">> => fun erlang:is_binary/1,
+                <<"hotspot">> => #{
+                    <<"id">> => erlang:list_to_binary(libp2p_crypto:bin_to_b58(PubKeyBin)),
+                    <<"name">> => erlang:list_to_binary(HotspotName),
+                    <<"rssi">> => 0.0,
+                    <<"snr">> => 0.0,
+                    <<"spreading">> => <<"SF8BW125">>,
+                    <<"frequency">> => fun erlang:is_float/1,
+                    <<"channel">> => fun erlang:is_number/1,
+                    <<"lat">> => fun erlang:is_float/1,
+                    <<"long">> => fun erlang:is_float/1
+                },
+                <<"mac">> => [
+                    #{
+                        <<"command">> => <<"link_adr_ans">>,
+                        <<"channel_mask_ack">> => 1,
+                        <<"data_rate_ack">> => 1,
+                        <<"power_ack">> => 1
+                    }
+                ],
+                <<"hold_time">> => fun erlang:is_integer/1
+            }
+        }
+    ),
+
+    {ok, #{<<"reported_at">> := DownlinkReportedAt1}} = test_utils:wait_for_console_event_sub(
+        <<"uplink_integration_req">>,
+        #{
+            <<"id">> => UplinkUUID1,
+            <<"category">> => <<"uplink">>,
+            <<"sub_category">> => <<"uplink_integration_req">>,
+            <<"description">> => erlang:list_to_binary(
+                io_lib:format("Request sent to ~p", [?CONSOLE_HTTP_CHANNEL_NAME])
+            ),
+            <<"reported_at">> => fun erlang:is_integer/1,
+            <<"device_id">> => ?CONSOLE_DEVICE_ID,
+            <<"data">> => #{
+                <<"req">> => #{
+                    <<"method">> => <<"POST">>,
+                    <<"url">> => <<?CONSOLE_URL/binary, "/channel">>,
+                    <<"body">> => fun erlang:is_binary/1,
+                    <<"headers">> => fun erlang:is_map/1,
+                    <<"url_params">> => fun test_utils:is_jsx_encoded_map/1
+                },
+                <<"integration">> => #{
+                    <<"id">> => ?CONSOLE_HTTP_CHANNEL_ID,
+                    <<"name">> => ?CONSOLE_HTTP_CHANNEL_NAME,
+                    <<"status">> => <<"success">>
+                }
+            }
+        }
+    ),
+
+    Time1 = DownlinkReportedAt1 - UplinkReportedAt1,
+    ?assert(
+        Time1 < DefaultFrameTimeout,
+        "No downlink to be sent, adr not acknowledged, still short window"
+    ),
+
+    ok = test_utils:ignore_messages(),
+
+    %% ========================================================================
+    %% Sending the third packet [fcnt: 2], just a basic packet. Nothing is going
+    %% down the device, so we use the uplink_integration_req event to do our
+    %% frame timeout calculation. By this time the device has recognized that
+    %% it's channels haven been corrected, and it should use a longer frame
```

Comment:
```suggestion
    %% its channels haven been corrected, and it should use a longer frame
```

Reply:
Looks like this was already fixed in not a suggestion